src/core/agent/services/keriaNotificationService.ts

```diff
@@ -959,10 +959,35 @@ class KeriaNotificationService extends AgentService {
         });
         break;
       }
+      case OperationPendingRecordType.Individual: {
+        const newRecordId = recordId.split(".")[1]
+        await this.identifierStorage.updateIdentifierMetadata(newRecordId, {
+          isPending: false,
+        });
+
+        const individualIdentifier =
+          await this.identifierStorage.getIdentifierMetadata(newRecordId);
+        await this.props.signifyClient.identifiers()
+          .addEndRole(individualIdentifier.id, "agent", this.props.signifyClient.agent!.pre);
```

Comment:
Can you test what happens when you try to do this twice in a row? Is it idempotent or causes issues?